test/JDBC/input/storedProcedures/Test-sp_rename-vu-verify.sql

```diff
@@ -0,0 +1,114 @@
+-- tsql
+
+USE master
+GO
+
+SELECT * FROM information_schema.tables
+GO
+
+SELECT schema_name, object_name FROM sys.babelfish_view_def
```

Comment:
Please use ORDER BY clause for SELECT statements, otherwise the output can be in different order